python/pyarrow/dataset.py

```diff
@@ -798,6 +799,22 @@ def write_dataset(data, base_dir, basename_template=None, format=None,
 
             def file_visitor(written_file):
                 visited_paths.append(written_file.path)
+    existing_data_behavior : 'error' | 'overwrite_or_ignore' |
+                             'delete_matching'
```

Comment:
```suggestion
    existing_data_behavior : 'error' | 'overwrite_or_ignore' | \
'delete_matching'
```

(I know this is ugly in the source code, but that's how to get it looking properly in the sphinx docs (the `name : type` should be a single line for rendering purposes)